.github/workflows/expo_cd.yml

```
@@ -1,26 +1,93 @@
+          fetch-depth: 0 # Full history for git push
+          token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Get commit info
         run: |
           echo "COMMIT_HASH=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
           echo "COMMIT_MSG=$(git log -1 --pretty=format:'%s')" >> $GITHUB_ENV
 
+      - name: Update version.properties
+        if: ${{ github.event.inputs.version_bump_type != 'None' }}
+        id: version_bump
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:31:46: Double quote to prevent globbing and word splitting

(shellcheck)